### PR TITLE
LPS-138477 Ignore usages of A.Modal

### PR DESCRIPTION
**Pull request by kresimir-coko**
This one's pretty simple as the usages are only found in 2 files, a deprecated one, and one that relies on `A.Modal` by extending it. I also took this opportunity to ignore other rules that I found inside the deprecated file, as it would have to be done eventually, so why not all at once in this simple PR.

### CODE REVIEW

**Comment by kresimir-coko** [comment]
Hey @bryceosterhaus I've combined your changes with mine, confirmed SF locally and force-pushed it. Should be ready for review

File: modules/apps/frontend-js/frontend-js-aui-web/src/main/resources/META-INF/resources/liferay/preview.js [hunk 1]
```diff
@@ -157,13 +164,14 @@ AUI.add(
 				},
 
 				_getMaxOverlay() {
+					alert('_getMaxOverlay');
```
[the remark this answers is not in the document]
Comment (bryceosterhaus) [reply modules/apps/frontend-js/frontend-js-aui-web/src/main/resources/META-INF/resources/liferay/preview.js:167]: ^ I left this comment probably early last week but never hit "submit" 😂

**Comment by bryceosterhaus** [comment]
Sent follow up at https://github.com/liferay-frontend/liferay-portal/pull/1549

File: modules/apps/frontend-js/frontend-js-aui-web/src/main/resources/META-INF/resources/liferay/preview.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (kresimir-coko) [reply modules/apps/frontend-js/frontend-js-aui-web/src/main/resources/META-INF/resources/liferay/preview.js:167]: Ah yea, got overlooked somehow

**Pull request closed without merging**